R/zchunk_L2252.land_input_5_irr_mgmt.R

```diff
@@ -1,34 +1,51 @@
 #' module_aglu_L2252.land_input_5_irr_mgmt
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces L2252.LN5_Logit, L2252.LN5_HistMgdAllocation_crop, L2252.LN5_MgdAllocation_crop,
```

Comment:
Maybe a little English too..."This chunk produces the inputs for the lowest level of the land nest, including disaggregated crop technologies."